ggshield/core/scan/file.py

```diff
@@ -68,6 +69,9 @@ def get_files_from_paths(
     :param paths: List of file/dir paths from the command
     :param recursive: Recursive option
     :param yes: Skip confirmation option
+    :param display_scanned_files: Display all scanned files in path (only for Secrets, for IaC/SCA
```

Comment:
I'm not fan of coding the current state of usage in the comment, as this may change later.
Maybe be less specific: *In some parts of the code (e.g. SCA), we might want to display a processed list instead*